### ISSUE TITLE
Using "include REXML" cause an exception error with "text" widjet

### ISSUE THREAD

**Issue by silkPK** [AndyObtiva/glimmer-dsl-swt#33]
**Desktop (please complete the following information and make sure it matches [Glimmer Pre-Requisites](https://github.com/AndyObtiva/glimmer-dsl-swt#pre-requisites) before reporting an issue):**
 - CPU Architecture [x86_64]
 - OS [Linux Ubuntu 22.04 LTS]:
 - Java Version [JDK 19.0.2]:
 - JRuby Version [jruby-9.3.10.0]:
 - Glimmer Libraries/Versions [glimmer 2.7.3 / glimmer-dsl-swt 4.27.0.1]:

**Describe the issue**
If the source code include REXML and text widget,  the execution throw an exception:

```
E, [2023-05-31T12:31:24.452499 #60765] ERROR -- : Error encountered in painting shape (#<Glimmer::SWT::Custom::Shape::Text:0x6a2badb1 args=[nil, :default, :default, true] properties={"foreground"=>[#<Java::OrgEclipseSwtGraphics::Color:0x18faaff6>], "font"=>[#<Java::OrgEclipseSwtGraphics::Font:0x137a4b54>]}} shapes=["#<Glimmer::SWT::Custom::Shape::Text:0x4ca970d5 args=[\"test\"] properties={}}>"]>) with method (drawText) calculated args () and args ([nil, :default, :default, true])
E, [2023-05-31T14:11:11.015709 #90968] ERROR -- : org.eclipse.swt.SWT.error(org/eclipse/swt/SWT.java:4899): Argument cannot be null (Java::JavaLang::IllegalArgumentException)
[...]
```

example code:
```
require 'glimmer-dsl-swt'
require 'rexml/document'

include Glimmer
include REXML

shell {
  text { text 'test' }
}.open
```

**Comment by AndyObtiva** [comment]
Glimmer DSL for SWT was not designed to have `REXML` mixed into the same namespace as `Glimmer`. What you did caused a namespace conflict. If you avoid `include REXML` and you use `REXML` another way, the code would work.

Also, the example you provided follows the way short demos are written, but it is not the recommended way for writing actual apps as it does not shield the public namespace from namespace pollution.

You're supposed to write an app like this:

```ruby
require 'glimmer-dsl-swt'

class TestApp
  include Glimmer::UI::CustomShell

  body {
    shell {
      text { text 'test' }
    }
  }
end

TestApp.launch
```

Afterwards, you could include `REXML` in a separate class, and have this `TestApp` class call the other class for any XML related work. That would prevent namespace pollution completely and any errors resulting from it.

I am closing this issue given that I provided an answer for what is causing it. If you have further questions or comments, feel free to respond to the closed issue, and I will see your comments (regardless of being closed).

Cheers.

**Comment by silkPK** [comment]
thanks for the exhaustive answer. 
 I opened the issue because I spent about 3 hours figuring out where the problem was.
 I hope it will be useful to someone else.  
Currently the problem occurs in a more complex structure than the simple example given, which follows your advice, where the REXML module is loaded in a separate class.  the way to avoid having the exception is to use the full name <module name>::<class name> instead of use the shortcut "include REXML".
Thank you  for the great work!
